chaos_genius/views/data_source_view.py

```diff
@@ -723,7 +724,9 @@ def get_table_info():
 
         ds_name = getattr(ds_data, "connection_type")
 
-        schema = None if SCHEMAS_AVAILABLE[ds_name] == False else schema
+        schema = (
+            None if ds_data.is_third_party or SCHEMAS_AVAILABLE[ds_name] == False else schema
```

Comment:
Would be slightly more readable to put parentheses around the condition:
```suggestion
            None if (ds_data.is_third_party or not SCHEMAS_AVAILABLE[ds_name]) else schema
```